### DISCUSSION TITLE
Great performance improvements recently!

### DISCUSSION THREAD

**Discussion by RustyBrakes** [JanDeDobbeleer/oh-my-posh#3650] [General]
I just wanted to say thank you for all your continued work on this fantastic tool. I feel like a month or so ago, my profile was taking 12 seconds to load and it's now down to 1 second! (this may be Windows Terminal improvements in Win11 or something, but I suspect it's probably Jan's tireless work)

![image](https://user-images.githubusercontent.com/6842623/228497495-a5ff4d73-0c3e-4e77-b45e-a9f4679a21fe.png)

**Comment by JanDeDobbeleer** [comment]
@RustyBrakes PowerShell 7.3 did quite some work to improve startup time, so I suspect they're mostly responsible for this. If they can fix `Invoke-Expression` execution time, that will make this lightning fast just like all other shells.
